http/server.lua

```diff
@@ -962,6 +1000,9 @@ local function httpd_stop(self)
         self.tcp_server:close()
         self.tcp_server = nil
     end
+
+    connections = {}
```

Comment:
It looks it should be a property of sever object. As I understand currently stop of one server will affect other servers (user could create several servers).

Reply:
Done.

```lua
diff --git a/http/server.lua b/http/server.lua
index 84f0f24..6b58f9d 100644
--- a/http/server.lua
+++ b/http/server.lua
@@ -16,10 +16,6 @@ local errno = require 'errno'
 
 local DETACHED = 101
 
--- A table that maps fiber object to time when it was active last time.
--- Using for tracking idle connections.
-local connections = {}
-
 local function close_idle_connections(connections, idle_timeout)
     -- FIXME: complexity is O(N) and it is bad
     for fiber_obj, start_time in pairs(connections) do
@@ -775,7 +771,7 @@ local function process_client(self, s, peer)
 
         if self.options.idle_timeout > 0 then
             connections[fiber.id] = fiber.clock()
-            close_idle_connections(connections, self.options.idle_timeout)
+            close_idle_connections(self.connections, self.options.idle_timeout)
         end
 
         local is_eof = false
@@ -1001,7 +997,7 @@ local function httpd_stop(self)
         self.tcp_server = nil
     end
 
-    connections = {}
+    self.connections = {}
 
     return self
 end
@@ -1387,6 +1383,10 @@ local exports = {
                 ctx         = {},
                 static      = {},
             },
+
+            -- A table that maps fiber object to time when it was active last
+            -- time. Using for tracking idle connections.
+            connections = {},
         }
 
         return self
```